### PR TITLE
Implement #530

### LINKED ISSUE #530: `c` doesn't do substring matching
# [Vyxal](https://github.com/Vyxal/Vyxal), 14 bytes
```
`Hello` `lo` c
```
[Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJgSGVsbG9gIGBsb2AgYyIsIiIsIiJd)

### PR DESCRIPTION
**Pull request by Lyxal**
Closes #530

### CODE REVIEW

**Review by chunkybanana — APPROVED** [review]

File: vyxal/elements.py [hunk 1]
```diff
@@ -561,11 +561,13 @@ def contains(lhs, rhs, ctx):
...
-    for item in lhs:
-        if item == rhs:
-            return 1
-    return 0
+    if list in vy_type(lhs, rhs, simple=True):
+        lhs = iterable(lhs, ctx=ctx)
+        for item in lhs:
+            if item == rhs:
+                return 1
+        return 0
```
Comment (pxeger) [inline vyxal/elements.py:569]: Just let `in` do the heavy lifting; it's completely equivalent.
```suggestion
        return int(lhs in rhs)
```

**Review by pxeger — APPROVED** [review]

**Comment by pxeger** [comment]
Oops, I didn't mean to approve. Now it's automerged...

File: vyxal/elements.py (hunk 1 above)
Comment (Lyxal) [reply vyxal/elements.py:569]: If it accounted for type overloading and infinite lazylists, I would. But it doesn't.

**Comment by pxeger** [comment]
#540

**Pull request merged**